src/seedu/addressbook/AddressBook.java

```diff
@@ -256,20 +272,20 @@ private static void echoUserCommand(String userCommand) {
      *
      * @param args full program arguments passed to application main method
      */
-    private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
-            showToUser(MESSAGE_INVALID_PROGRAM_ARGS);
-            exitProgram();
-        }
-
-        if (args.length == 1) {
-            setupGivenFileForStorage(args[0]);
-        }
-
-        if(args.length == 0) {
-            setupDefaultFileForStorage();
-        }
-    }
+//    private static void processProgramArgs(String[] args) {
```

Comment:
avoid leaving methods commented out. since we're using version control, you can easily get this method back by looking at the commit history.